django_project/certification/models/course_convener.py

```diff
@@ -0,0 +1,60 @@
+# coding=utf-8
+"""
+Course convener model definitions for certification apps
+"""
+
+from django.core.urlresolvers import reverse
+from django.utils.text import slugify
+from core.settings.contrib import STOP_WORDS
+from django.db import models
+from course import Course
+from unidecode import unidecode
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+class CourseConvener(models.Model):
+    """Course Convener model."""
+
+    name = models.CharField(
+        help_text="Course convener name",
+        max_length=250,
+        null=False,
+        blank=False,
+    )
+
+    email = models.CharField(
+        help_text="Course convener email",
+        max_length=150,
+        null=True,
+        blank=True,
+    )
+
+    slug = models.SlugField()
+    objects = models.Manager()
+    course = models.ManyToManyField(Course)
```

Comment:
course_convener has Many course, but one course can be own just by one course_convener (one-to-many). So it should be in course, it has course_convener  foreignkey and remove this field.